pkg/cluster/manager/display.go

```diff
@@ -254,6 +261,23 @@ func (m *Manager) Display(name string, opt operator.Options) error {
 	return nil
 }
 
+func getGrafanaURLStr(clusterInstInfos []InstInfo) (result string, exist bool) {
+	var grafanaURL []string
+	for _, instance := range clusterInstInfos {
+		if instance.Role == "grafana" {
+			grafanaURL = append(grafanaURL, fmt.Sprintf("%s:%d", instance.Host, instance.Port))
+		}
+	}
+	if len(grafanaURL) == 0 {
+		return "", false
+	}
+	grafanaURLs := fmt.Sprintf("http://%s", grafanaURL[0])
+	for i := 1; i < len(grafanaURL); i++ {
```

Comment:
This could be simplified by inserting `http://%s:%d` in L268, and use a `strings.Join()` here to concat them.

Reply:
done